app/views/admin/submissions/_submission_details.html.erb

```diff
@@ -3,7 +3,7 @@
     Artist/Designer
   </div>
   <div class='overview-item-value'>
-    <%= @artist&.[](:name)&.upcase %>
+    <a class="make-primary-link" href="<%= artist_url(@artist&.[](:slug)) %>"><%= @artist&.[](:name)&.upcase %></a>
```

Comment:
**nitpick**: is there a reason to not use [link_to](https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to) from rails?

Reply:
Ahh, no. Good call. Will make that change 👍🏼